### DISCUSSION TITLE
Battleground Games don't spawn food after initial spawns

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#195] [Bug Reports]
I know this is brand new, so sorry if this is known and I'm just adding noise!

I noticed that both the "Standard" and "Wrapped" battlegrounds don't seem to be spawning food after the initial food. This leads to the game ending at just over 100 turns since all the snakes eventually starve out

Sample Wrapped Game: https://play.battlesnake.com/g/267f3c98-6cd0-451f-b514-d65da90c85af/
Sample Standard Game: https://play.battlesnake.com/g/d410605b-715b-4f15-af47-c6b60485d707/

**Comment by bvanvugt** [comment]
We're on it, and very much appreciate the report! Thanks 👍

**Comment by bvanvugt** [comment]
@coreyja Believe this is fixed now, let me know if you see any other oddities.
